src/border_agent/border_agent.cpp

```diff
@@ -111,13 +111,15 @@ struct StateBitmap
     uint32_t mAvailability : 2;
     uint32_t mBbrIsActive : 1;
     uint32_t mBbrIsPrimary : 1;
+    uint32_t mEPSKcSupported : 1;
```

Comment:
Suggest name change from `mEPSKcSupported -> mEpskcSupported`.

Our naming style uses `CamelCase` and generally applies this to abbreviations as well (treat them as a noun). e.g., `mBbrIsPrimary ` for BBR (back-bone router).

Reply:
Thanks! Accept.